docker/transformers-pytorch-deepspeed-latest-gpu/Dockerfile

```diff
@@ -8,6 +8,9 @@ ARG PYTORCH='2.8.0'
 # Example: `cu102`, `cu113`, etc.
 ARG CUDA='cu126'
 
+# This needs to be compatible with the above `PYTORCH`. Let's use cpu version for now.
+ARG TORCHCODEC='0.7.0'
```

Comment:
previously (around 2025/07), this line likely had `soundfile` and being replaced by `torchcodec`.

If you prefer to simply remove `torchcodec` in this docker file, let me know.